test/path_filtering.go

```diff
@@ -98,3 +100,9 @@ func stablePathFilter(p string) bool {
 func alphaPathFilter(p string) bool {
 	return strings.Contains(p, "/alpha/") || strings.Contains(p, "/beta/") || stablePathFilter(p)
 }
+
+// betaPathFilter returns true for any example that should be allowed to run
+// when "enable-api-fields" is "beta".
+func betaPathFilter(p string) bool {
```

Comment:
I have been trying to understand if there are any thing we need to do with the stable Path? Previously we regard `beta` as stable now that we have `v1` which is stable, otherwise it feels the beta testing just a union of stable tests?

Reply:
I think:
`Alpha`: tests all examples inside `v1beta1/, v1beta1/alpha, v1beta1/beta` (basically everything)
`Beta`: tests all examples inside `v1beta1/ and v1beta1/beta` but not `v1beta1/alpha` (skip examples in alpha directory).
`Stable`: tests examples inside `v1beta1` (skip the `alpha` and `beta` folders).

Reply:
So I don't think we need to make any changes to stablePath or alphaPath.

Reply:
I think previously `beta` is `stable` according to
https://github.com/tektoncd/community/blob/main/teps/0033-tekton-feature-gates.md#new-api-fields. But now that `GA` is stable, `beta` should not be in `stablePath`?

Reply:
I was confused by that for a while too but if you look carefully at the stable path it's filtering on 

**!(** string.contains(alpha) || string.contains(beta) **)**

So its picking only the standard ones.

Reply:
Ahhhhh 👓 was in my shopping cart.
Thanks.

Reply:
Ohhh this is confusing 😵‍💫, wondering it would be better if we change the condition to !string.contains(alpha) && !string.contains(beta)